src/modules/esl-tooltip/core/esl-tooltip.ts

```diff
@@ -87,6 +105,22 @@ export class ESLTooltip extends ESLPopup {
     }
   }
 
+  @bind
+  protected _onKeyboardEvent(e: KeyboardEvent) {
+    super._onKeyboardEvent(e);
+    if (e.key === TAB) this._onTabKey(e);
+  }
+
+  protected _onTabKey(e: KeyboardEvent) {
+    if (this.activator &&
+        (!this.lastFocusableElement || e.target === this.lastFocusableElement) &&
+        !e.shiftKey) {
+      this.activator.focus();
+      e.stopPropagation();
+      e.preventDefault();
+    }
```

Comment:
```suggestion
    if (!this.activator) return;
    const {lastFocusableElement} = this;
    if ((!lastFocusableElement || e.target === lastFocusableElement) && !e.shiftKey) {
      this.activator.focus();
      e.stopPropagation();
      e.preventDefault();
    }
```
To calculate this.lastFocusableElement once.